components/reply.js

```diff
@@ -115,38 +140,38 @@ export default function Reply ({ item, onSuccess, replyOpen, children, placehold
             {/* HACK if we need more items, we should probably do a comment toolbar */}
             {children}
           </div>)}
-      {reply &&
-        <div className={styles.reply}>
-          <Form
-            initial={{
-              text: ''
-            }}
-            schema={commentSchema}
-            invoiceable
-            onSubmit={onSubmit}
-            storageKeyPrefix={'reply-' + parentId}
-          >
-            <MarkdownInput
-              name='text'
-              minRows={6}
-              autoFocus={!replyOpen}
-              required
-              placeholder={placeholder}
-              hint={me?.sats < 1 && <FreebieDialog />}
-              innerRef={replyInput}
-            />
-            {reply &&
-              <div className='mt-1'>
-                <FeeButton
-                  baseFee={1} parentId={parentId} text='reply'
-                  ChildButton={SubmitButton} variant='secondary' alwaysShow
-                />
-              </div>}
-          </Form>
-        </div>}
+      <div className={styles.reply} style={{ display: reply ? 'block' : 'none' }}>
```

Comment:
Instead of conditionally rendering this div, I am toggling `display` CSS because I need the `Form` to exist even when the reply isn't shown, for the ref to be used.